### PR TITLE
[EN-714] added new translations for producer invite and fixed minor issues

### PR DESCRIPTION
**Pull request by James-FarmerJoe**
James can slam revolving doors.

### CODE REVIEW

File: languages/ar/node-farmerjoe.json [hunk 1]
```diff
@@ -3,20 +3,21 @@
...
-      "greetingPart3": "للعمل معًا بشكل أفضل في المستقبل ، يريد {{name}} إضافتك كـ {{role}}.",
+      "greetingPart2": "أود أن أدعوكم إلى المزارع جو",
+      "greetingPart3": "لكي نتمكن من العمل معًا بشكل أفضل في المستقبل ، يود {{name}} إضافتك إلى FarmerJoe كـ {{role}}.",
       "greetingPart4": "يمكنك رسم مناطق الزراعة الخاصة بك وتقديمها في تطبيق FarmerJoe. هذا يجعل العمليات والتواصل أسهل ، لذلك يمكن لشركة {{company}} تقديم المشورة والدعم لك بشكل أفضل.",
       "installInstructionsTitle": "وإليك كيف يعمل:",
-      "installInstructionsPart1": "قم بتنزيل تطبيق FarmerJoe إذا لم تكن قد قمت بذلك بالفعل",
-      "installInstructionsPart2": "سجل باستخدام عنوان بريدك الإلكتروني {{email}} في farmerJoe",
-      "installInstructionsPart3": "سوف تحصل على تأكيد عبر البريد الإلكتروني لتفعيل حسابك",
+      "installInstructionsPart1": "قم بتنزيل تطبيق FarmerJoe إذا لم يكن لديك بالفعل",
+      "installInstructionsPart2": "التسجيل مع FarmerJoe باستخدام {{email}}",
```
Comment (cawocarl) [inline languages/ar/node-farmerjoe.json:11]: So it's correct that we've used PascalCase for _FarmerJoe_  in Arabic and camelCase in other languages?

Comment (James-FarmerJoe) [reply languages/ar/node-farmerjoe.json:11]: where do you see pascal case here?

Comment (James-FarmerJoe) [reply languages/ar/node-farmerjoe.json:11]: ah I see now, no thats due to the language service we use

File: languages/es/node-farmerjoe.json [hunk 2]
```diff
@@ -3,20 +3,21 @@
     "invitations": {
       "title": "Invitación",
       "greetingPart1": "Hola {{invitee}},",
-      "greetingPart2": "te invita a unirte a farmerJoe.",
-      "greetingPart3": "Para trabajar mejor juntos en el futuro, {{name}} quiere agregarte como {{role}}.",
+      "greetingPart2": "Me gustaría invitarte a FarmerJoe.",
```
Comment (cawocarl) [inline languages/es/node-farmerjoe.json:6]: Inconsistent use of PascalCase and camelCase with the word _FarmerJoe_ in this language.

File: languages/cs/node-farmerjoe.json [hunk 3]
```diff
@@ -3,20 +3,21 @@
     "invitations": {
       "title": "Pozvání",
       "greetingPart1": "Ahoj {{invitee}},",
-      "greetingPart2": "vás zve, abyste se připojili k farmáři Joe.",
-      "greetingPart3": "Chcete-li v budoucnu lépe spolupracovat, {{name}} vás chce přidat jako {{role}}.",
+      "greetingPart2": "rád bych vás pozval do farmářeJoe",
```
Comment (James-FarmerJoe) [inline languages/cs/node-farmerjoe.json:6]: @cawocarl & @fj-vp this happens alot that names or certain words including our placeholders get replaced. We need to replace all non-translatable placeholders to non human readable such as %$0 or something

**Review by cawocarl — APPROVED** [review]

File: languages/ar/node-farmerjoe.json (hunk 1 above)
Comment (sirnornur) [reply languages/ar/node-farmerjoe.json:11]: I also vote for consistency. I guess we had discussed this topic in a meeting. 
We could add simple script to the build process that replaces `farmerjoe` (case-insensitive) to the form we want.
James, can you try it out?

**Review by sirnornur — APPROVED** [review]

**Review by fj-vp — APPROVED** [review]

**Pull request merged**